packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -192,14 +194,25 @@ export class GarbageCollector implements IGarbageCollector {
 			const overrideSessionExpiryTimeoutMs = this.mc.config.getNumber(
 				"Fluid.GarbageCollection.TestOverride.SessionExpiryMs",
 			);
-			const timeoutMs = overrideSessionExpiryTimeoutMs ?? this.configs.sessionExpiryTimeoutMs;
+			let timeoutMs = this.configs.sessionExpiryTimeoutMs;
 
+			if (pendingSessionExpiryTimerStarted) {
+				const timeLapsedSincePendingTimer = Date.now() - pendingSessionExpiryTimerStarted;
```

Comment:
```suggestion
				// NOTE: This assumes the client clock hasn't been tampered with since the original session
				const timeLapsedSincePendingTimer = Date.now() - pendingSessionExpiryTimerStarted;
```